### ISSUE TITLE
How to train with Indian Accent

### ISSUE THREAD

**Issue by Jainu-s** [ming024/FastSpeech2#227]
Hi, your repo was very much helpful for TTS. I want to train model with custom Indian voice as I have Indian voice (.wav) dataset along with transcription.  I want to train custom model with single voice will this repo capable of doing it? because I don't have any Indian phonotics so still can I achieve Indian accent with the dataset I have? can you please help me?

Thank You

**Comment by Lakhjeet1082** [comment]
yes you can do that for Indian language. You can use MFA to create your phonetic dictionary and TextGrids or if pre-trained dictionary for your language is not available you can use phonemizer to make the dictionary.Then train your acoustic model using MFA and align the data to generate TextGrids. And then you can go ahead with the repo for further procedure

**Comment by Lakhjeet1082** [comment]
Use mfa or phonemizer to generate pronounciation dictionary .Then validate your corpus and check for OOV words using ( mfa validate ~/mfa_data/my_corpus ~/mfa_data/my_dictionary.txt ) this command. If the pretrained acoustic model for your language is not available then use ( mfa train ~/mfa_data/my_corpus ~/mfa_data/my_dictionary.txt ~/mfa_data/new_acoustic_model.zip ) use this command to train acoustic model. Then use (mfa align ~/mfa_data/my_corpus english_us_arpa english_us_arpa ~/mfa_data/my_corpus_aligned) mfa align command to generate textgrids. And then you are ready to use Fastspeech2 . For more info you can refer https://montreal-forced-aligner.readthedocs.io/en/latest/first_steps/index.html this documentation
